src/js/on-demand-scripts/qg-load-google-api.js

```diff
@@ -0,0 +1,148 @@
+(function (qg, $) {
```

Comment:
Isn't this file in the qg-web-template / web-template-release?

Reply:
probably this file is not used on any matrix page, for now i didn't make any modification on the src file and just copy everything over, can be checked and removed by manjeet after the infrastructure is finalised (in seperate PR)